battlemap_processor/core/image_source_handler.py

```
@@ -92,54 +116,65 @@ def __init__(self, temp_dir: Optional[str] = None, request_timeout: int = 30):
+    def _safe_extract_member(
+        self, zip_ref: zipfile.ZipFile, member: zipfile.ZipInfo, extract_dir: str
+    ) -> str:
         """Safely extract a zip member, preventing zip-slip attacks"""
         # Normalize the member path
         member_path = os.path.normpath(member.filename)
-        
+
         # Check for directory traversal attempts
-        if member_path.startswith('/') or '..' in member_path:
+        if member_path.startswith("/") or ".." in member_path:
             raise UnsafeZipError(f"Unsafe path in zip: {member.filename}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)